module/PowerNSX.psm1

```diff
@@ -9861,13 +9875,20 @@ function New-NsxLogicalSwitch  {
     the Transport Zone it is created in, but CP mode can specified as required.
 
     .EXAMPLE
+    Get-NsxTransportZone | New-NsxLogicalSwitch -name LS6
+
+    Create a Logical Switch with default control plane mode on all Transport Zone.
+
+    .EXAMPLE
+    Get-NsxTransportZone -LocalOnly | New-NsxLogicalSwitch -name LS6
+
+    Create a Logical Switch with default control plane mode on Local Transport Zone.
```

Comment:
This command will create LS6 on ALL locally configured transport zones. Need to either specify a transport zone name and pipe to new-nsxlogicalswitch, or amend the description to specify that this will create the logical switch on ALL Local Transport Zones